python/txt2tags.py

```diff
@@ -2206,8 +2217,8 @@ def parse(self):
 		raw = self.get_target_raw()
 		for target, key, value in raw:
 			self.add(key, value)
-		Message(_("Added the following keys: %s")%string.join(
-		         self.parsed.keys(),', '),2)
+		Message(_("Added the following keys: %s")% ', '.join(
+		         list(self.parsed.keys())),2)
```

Comment:
Makes sense to join the keys here, but you do not need to wrap that in a list I believe. Join will simply iterate over the keys